pkg/controllers/dynakube/dynakube_controller.go

```diff
@@ -118,12 +119,20 @@ func (controller *Controller) Reconcile(ctx context.Context, request reconcile.R
 
 	dynaKube, err := controller.getDynakubeOrUnmap(ctx, request.Name, request.Namespace)
 	if err != nil {
+		log.Info("reconciling DynaKube finished, could not get dynakube", "namespace", request.Namespace, "name", request.Name, "result", "empty", "err", err.Error())
```

Comment:
I added some logs to clearly see which part of the reconciliation has been started and finshed when. This can be really helpful when analysing timing related issues between processes (e.g. operator vs. webhook).

Reply:
my main concern is/was the possible duplicated logging of errors, the rest are fine by me